src/Init/Data/Int/Order.lean

```diff
@@ -799,6 +804,12 @@ protected theorem lt_add_of_neg_lt_sub_right {a b c : Int} (h : -b < a - c) : c
 protected theorem neg_lt_sub_right_of_lt_add {a b c : Int} (h : c < a + b) : -b < a - c :=
   Int.lt_sub_left_of_add_lt (Int.sub_right_lt_of_lt_add h)
 
+protected theorem add_lt_iff (a b c : Int) : a + b < c ↔ a < -b + c := by
+  rw [← Int.add_lt_add_iff_left (-b), Int.add_comm (-b), Int.add_neg_cancel_right]
+
+protected theorem sub_lt_iff (a b c : Int) : a - b < c ↔ a < b + c :=
```

Comment:
Here I would have expected
```suggestion
protected theorem sub_lt_iff (a b c : Int) : a - b < c ↔ a < c + b :=
```

Reply:
Easy enough to fix the proof, so sure.

There are way too many lemmas in this file, given that we now have `omega`, but that's a story for another day.